samples/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This sample application provides an example usage of the display ILI9163C from ILITEK.
+The example includes PWM control for backlight dimming and displays text and a counter on the screen.
+
+- Displays "6TRON BY CATIE!" on the screen. 
+- Shows an incrementing counter at the bottom of the screen. 
+- Uses PWM to control the backlight brightness in a pulsing pattern. 
+- Refreshes the display content periodically using LittlevGL (LVGL).
+
+> [!NOTE]
+>  Ensure the display and PWM hardware configurations match the setup below for correct operation.
```

Comment:
Might be good to add an example  for the pwm setup you have right now. It is not ubiquitus to everyone how to setup the pwm correctly for the driver.

Reply:
We must ask to @clementlgl